README.md

```diff
@@ -19,3 +87,58 @@ TOKEN_URI="value"
 AUTH_PROVIDER_X509_CERT_URL="value"
 CLIENT_X509_CERT_URL="value"
 ```
+
+#### Github Access Token
+
+If you intend to use GridGopher's GitHub interactions features, it's required to
+create a GitHub access token and place it correctly in the project repository.
+To create a token, please use the official guide found
+[here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+Created tokens should be granted `repo` access.
+
+Once a token has been created, there are two ways to allow GridGopher
+to use it:
+
+1. Create a json file in the repository root with the following format (note
+   that the `gh_access_token` key is required):
+
+   ```json
+   {
+     "gh_access_token": "paste your token here"
+   }
+   ```
+
+1. OR add a variable to the `.env` file in the following format
+
+     ```.env
+     GH_ACCESS_TOKEN="paste your token here"
+     ```
+
+### Writing Config
+
+GridGopher relies on user written YAML configuration to collect data from Google
+Sheets and organize it in regions. GitHub interactions are also managed by
+YAML configuration. To read more about the structure of GridGopher
+configuration, please refer to our [schema documentation](docs/schemas.md).
+
+#### Sheets Collector
+
+Sheets Collector the component of GridGopher responsible for making Google
+Sheets API calls and retreiving data from user specified files and sheets.
+Additionally, it creates an object oriented structure for regions and sheets
+of data. In order to use this component, configuration YAML files are needed
+in the `config/sheet_sources` directory. Multiple files can be used if multiple
+sheets are being read at the same time.
+
+#### Github Interactions
+
+Another component of GridGopher is the GitHub Interaction interface. It's
+responsible for making API requests to GitHub and posting user specified
+information to GitHub in the form of Issue Trackers, Pull Requests, and files.
```

Comment:
Here the two terms are capitalized. Change or ignore this based on what it decided for uniformity.